### ISSUE TITLE
Remove forks from the main list

### ISSUE THREAD

**Issue by Paebbels** [berndca/discover_fossi#2]
Hello,

the list also contains forks. Is it possible to remove the forks from the main list and display them e.g. with a "plus" button?

Advantages:
- removes duplicates / noise
- correct the number of repositories

Example:

```
JimLewis/OSVVM
 o- VLSI-EDA/OSVVM
```

Hmmm starnge ... This list misses 6 other forks. So the question to me is: Either the list is a bit old (I don't think so) or you tried to remove forks, but didn't found all of them. What's the criteria to discover forks?

**But in general: GREAT WORK!**

Kind regards
    Patrick

**Comment by berndca** [comment]
Thanks for your feedback. The current database does indeed contain forks. I have left all forks in which had at least one star. Some forks are actually continuations of the original project which in some cases appear abandoned. I agree that grouping forked projects under the source would be a reasonable approach, but I'm concerned that it might make it more difficult to identify forked projects which are actively developed because the original project has gone dormant. Any suggestions?

Thanks, Bernd

**Comment by Paebbels** [comment]
Yes indeed, it's hard to distinguish if for example a fork is more active then the original repository ...
Can you give a number of forks, that are include in the listed 2.9 k repos?

![image](https://cloud.githubusercontent.com/assets/956109/19447434/c5146b96-949d-11e6-9c18-37ebc5f3b6ea.png)

I would like to see forked repositories to be grouped under the original repositor (maybe with an indentation). They should appear in the same search result, because they share keywords, right?

Btw. Why has the second OSVVM entry no license set?

**Comment by berndca** [comment]
This project is based on the data provided by the Github api (the preview to be exact). The license does not show for the second OSVVM is because Github does not seem to know about it. If you look at the main header starting with commits, branches ... you'll find that there is a license shown in the JimLewis/OSVVM but not in VLSI-EDA/OSVVM.

Unfortunately the only info about forked repos in the data provided by the api is whether the repo is a fork or not. The api does support listing the forks (children) for a given repo. However I have not been able to find a way to identify the "parent"-repo of a forked one. At one point I created an index of all forked repos with all their "children" and used it look up the "parents". However this is rather brittle. If one would add a forked repo without having added it's "parent" first the automatic flow would fail.

Let me think about this.

Bernd
